### PR TITLE
Videos/podcast outline

### LINKED ISSUE #10: Outline for videos&podcast page

### PR DESCRIPTION
**Pull request by jgrgurovic**

### CODE REVIEW

**Comment by ilovric00** [comment]
@jgrgurovic build is failing

**Review by ilovric00 — CHANGES_REQUESTED** [review]
This PR contains some part from other PR. 

Let me explain you tomorrow how you should handle this.

**Comment by jgrgurovic** [comment]
> @jgrgurovic build is failing
Wrong path for hero image, build is fine now

**Comment by jgrgurovic** [comment]
> This PR contains some part from other PR.
> 
> Let me explain you tomorrow how you should handle this.
It got somehow mixed with the Login one , not sure how or why but sure we'll see tomorrow

File: package.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
     "postcss": "8.4.28",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "react-icons": "^4.11.0",
```
Comment (ilovric00) [inline package.json]: I think this is an overkill to you external lib just for the handling the icons

File: views/HeroSection/index.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+"use client"
+import React from "react"
+import Image from "next/image"
+import HeroImage from "../../public/hero_streetimage.png"
+import { usePathname } from "next/navigation"
+
+function capitalizeFirstLetter(string: string): string {
+  return string.charAt(0).toUpperCase() + string.slice(1)
+}
```
Comment (ilovric00) [inline views/HeroSection/index.tsx]: move this to `utils/static` folder

File: views/Search&FilterSection/SearchBar.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from "react"
+import { FaSearch } from "react-icons/fa"
+const SearchBar = () => {
```
Comment (ilovric00) [inline views/Search&FilterSection/SearchBar.tsx:4]: missing space before and after...do you use Prettier?

**Review by ilovric00 — APPROVED** [review]

**Pull request merged**